docs/content/documentation/content/shortcodes.md

```diff
@@ -169,3 +169,30 @@ Usage example:
 Result example:
 
 {{ gist(url="https://gist.github.com/Keats/e5fb6aad409f28721c0ba14161644c57") }}
+
+### Twitch
+Embed a Twitch stream with or without chat.
+
+The arguments are:
+
+- `channel`: the channel name (mandatory)
+- `chat`: shows the stream chat next to the video embed 
+- `class`: a class to add the `div` surrounding the iframe
+- `videoWidth`: the px **width** the video embed should have 
```

Comment:
standard is snake_case, not camelCased

Reply:
snake_case is ignored (another regex problem too?), we should create a new Issue for that

Reply:
It should be fixed on master